src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.props

```diff
@@ -51,8 +51,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DirectoryBuildPropsPath Condition="'$(_DirectoryBuildPropsBasePath)' != '' and '$(_DirectoryBuildPropsFile)' != ''">$([System.IO.Path]::Combine('$(_DirectoryBuildPropsBasePath)', '$(_DirectoryBuildPropsFile)'))</DirectoryBuildPropsPath>
   </PropertyGroup>
 
+  <Import Project="$(CustomBeforeDirectoryBuildProps)" Condition="'$(CustomBeforeDirectoryBuildProps)' != '' and exists('$(CustomBeforeDirectoryBuildProps)')" />
```

Comment:
Based on https://github.com/dotnet/sdk/issues/36749#issuecomment-1802667057, I think @dsplaisted meant for the `DirectoryBuildPropsPath` Import to be removed here, and the `CustomBeforeDirectoryBuildProps`/`CustomAfterDirectoryBuildProps` to get set to implement this logic through callbacks from `Microsoft.Common.props`.

Reply:
Correct.  We should remove this duplicated import logic instead of adding more to it.